src/app/pages/system/advanced/access/access-form/access-form.component.ts

```diff
@@ -29,15 +29,16 @@ export class AccessFormComponent implements OnInit {
   form = this.fb.group({
     token_lifetime: [defaultPreferences.lifetime, [
       Validators.required,
-      Validators.min(30),
-      Validators.max(Math.floor(2 ** 31 / 1000 - 1)), // Max value for setTimeout
+      Validators.min(30), // Min value allowed is 30 seconds.
+      Validators.max(2147482), // Max value is 2147482 seconds, or 24 days, 20 hours, 31 minutes, and 22 seconds.
```

Comment:
just curious, how is this fixing it? I tried it in the console, and the `Math.floor` function returns the same thing...

Reply:
> just curious, how is this fixing it? I tried it in the console, and the `Math.floor` function returns the same thing...

I don't know why do we need this calculation `Math.floor(2 ** 31 / 1000 - 1)` if documentation clearly says max is `2147482` (basically the same)

Reply:
Max for setTimeout is a 32bit value, hence 2 to the power of 32 - 1 second was used.

Reply:
Maybe good to have this documented with `Math.floor`, everybody was asking this in the forum...

Reply:
> Maybe good to have this documented with `Math.floor`, everybody was asking this in the forum...

@truenas/docs-team